__tests__/app.test.js

```diff
@@ -313,5 +312,79 @@ describe("Using app.js to run the database of NC-games", () => {
         });
     });
   });
+  describe("8.GET/api/reviews this can take a query!", () => {
+    test("200: response with a Array of objects with reviews", () => {
+      return request(app)
+        .get("/api/reviews")
+        .expect(200)
+        .then(({ body }) => {
+          expect(Array.isArray(body.reviewArray)).toBe(true);
```

Comment:
321 - we don't technically need this as using forEach on line 324 implies that body.reviewArray is an array

Reply:
Fair enough, I'll remove them